### DISCUSSION TITLE
Can go in reverse?

### DISCUSSION THREAD

**Discussion by wnadim92** [DontShaveTheYak/cf2tf#98] [Q&A]
Can this go from tf to cf?

**Comment by shadycuz** [comment]
This is something that I have thought about. It would be nice to take a well know Terraform module and dump it to Cloudformation. I'm not sure I would do it in this tool. Mainly because most of the nice Terraform tooling is all written in GO. So it might make more sense to do it in GO.

I'm also not sure I would do it for free and open-source it. Maybe a website where you could pay $5 and have it converted?

**Reply by SEGStriker** [reply]
I was looking for the same and suggested that there should be such request. So here I am with the same - how to convert TF to CF :)
